### DISCUSSION TITLE
armsr/armv7下使用image builder无法安装一些kmod（有完整日志）

### DISCUSSION THREAD

**Discussion by 14790897** [immortalwrt/immortalwrt#1516] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/646a4cc3-9c4a-49c8-b619-a72aca0dcff7)

https://github.com/14790897/OpenWrt-In-Docker/actions/runs/11337450115/job/31529076142

**Comment by 1715173329** [comment, marked as answer]
这些包不存在于 armsr/armv7
